src/containers/Team/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import {
+    Footer,
+    Header,
+} from '../../components'
+
+import TeamItem from '../../components/TeamItem/index';
+import { teamItems } from '../../constants/index';
+
+const Team = (item) => (
+    <div>
+        <div className="page-background">
+            <Header/>
+            <div className="team-header">
+                Team Carbon Doomsday
+            </div>
+            <div>
+                <div className="team-grid">
+                {teamItems.map((item) => (
```

Comment:
Here we should pass `teamItems` down as props. 

We could also take advantage of [default props](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#default-prop-values)

by doing something like 
````javascript
Team.defaultProps = {
teamItems: teamItems
};
````

And then within the team component pass `teamItems` as props.

The reason for this approach is that if anyone else were to reuse this component and wanted to pass their own team members props they could.

If the `teamItems` prop is not passed to the component then we just use the defaultProp which is teamItems from our constants file. 

So for us we would be using `Team` component without passing it any props

Reply:
It would also be of worth to add `proptypes` to this component